website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,306 @@
+  lightSrc: "@/assets/blog/roadmap-2024/banner-light.png"
+  darkSrc: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+Today we’re excited to announce the release of _Biome v1.7_!
+
+This new version provides an easy path to migrate from ESLint and Prettier.
+It also introduces experimental machine-readable reports for the formatter and the linter, new linter rules, and many fixes.
+
+If you’re not familiar with _Biome_ yet, it is a fast formatter and linter for JavaScript, TypeScript, JSX, and JSON that scores [97% compatibility with Prettier](https://console.algora.io/challenges/prettier) and provides [more than 200 linter rules](/linter/rules/).
+
+Update _Biome_ using the following commands:
+
+```
+npm install --save-dev --save-exact @biomejs/biome@latest
+npx @biomejs/biome migrate
+```
+
+## Migrate from _ESLint_ with a single command
+
+This release introduces a new command `biome migrate eslint`.
+This command will read your _ESLint_ configurations and attempt to port their settings to Biome.
+
+The command handles both the legacy and the flat _ESLint_ configurations.
+It supports the `extends` field of the legacy configuration and loads both shared and plugin configurations!
+The command also migrates `.eslintignore`.
+
+Given the following _ESLint_ configuration:
+
+```json title=".eslintrc.json"
+{
+  "extends": ["plugin:unicorn/recommended"],
+  "plugins": ["unicorn"],
+  "ignore_patterns": ["dist/**"],
+  "globals": {
+    "Global1": "readonly"
+  },
+  "rules": {
+    "eqeqeq": "error"
+  },
+  "overrides": [
+    {
+      "files": ["tests/**"],
+      "rules": {
+        "eqeqeq": "off"
+      }
+    }
+  ]
+}
+```
+
+And the following _Biome_ configuration (obtained by running `biome init`):
+
+```json title="biome.json"
+{
+	"organizeImports": {
+		"enabled": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	}
+}
+```
+
+Run `biome migrate eslint --write` to migrate your _ESLint_ configuration to _Biome_.
+This results in the following _Biome_ configuration:
+
+```json title="biome.json"
+{
+	"organizeImports": { "enabled": true },
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": false,
+			"complexity": {
+				"noForEach": "error",
+				"noStaticOnlyClass": "error",
+				"noUselessSwitchCase": "error",
+				"useFlatMap": "error"
+			},
+			"style": {
+				"noNegationElse": "off",
+				"useForOf": "error",
+				"useNodejsImportProtocol": "error",
+				"useNumberNamespace": "error"
+			},
+			"suspicious": {
+				"noDoubleEquals": "error",
+				"noThenProperty": "error",
+				"useIsArray": "error"
+			}
+		}
+	},
+	"javascript": { "globals": ["Global1"] },
+	"overrides": [
+		{
+			"include": ["tests/**"],
+			"linter": { "rules": { "suspicious": { "noDoubleEquals": "off" } } }
+		}
+	]
+}
+```
+
+Note that this overrides your initial _Biome_ configuration.
```

Comment:
This aspect of the logic should be part of the previous phrase, when you actually explain the example and present the new configuration file.

Also, I suggest to better explain what "override" means in this case, maybe by using different words (I don't understand what this means here, and how it connects to the example)